### PR TITLE
Controller updates

### PR DESCRIPTION
**Pull request by ywwg**
@Pegasus-RPG

### CODE REVIEW

**Comment by sblaisot** [comment]
Don't forget to add new files to Windows nsis uninstaller please.

**Comment by ywwg** [comment]
done, I think

File: res/controllers/Gemini CDMP-7000 L audio.midi.xml [hunk 1]
```diff
@@ -0,0 +1,477 @@
+<?xml version='1.0' encoding='utf-8'?>
+<MixxxControllerPreset mixxxVersion="" schemaVersion="1">
+    <info/>
+    <controller id="gemini">
+        <scriptfiles>
+            <file functionprefix="cdmp7000" filename="Gemini_CDMP-7000-scripts.js"/>
```
Comment (rryan) [inline res/controllers/Gemini CDMP-7000 L audio.midi.xml]: Could you rename the script file to have dashes?

Comment (ywwg) [inline res/controllers/Gemini CDMP-7000 L audio.midi.xml]: done

File: build/nsis/Mixxx.nsi [hunk 2]
```diff
@@ -397,6 +397,9 @@ Section "Uninstall"
   Delete "$INSTDIR\controllers\Electrix-Tweaker-scripts.js"
   Delete "$INSTDIR\controllers\Evolution_Xsession.midi.xml"
   Delete "$INSTDIR\controllers\FaderFoxDJ2.midi.xml"
+  Delete "$INSTDIR\controllers\Gemini CDMP-7000 L audio.midi.xml"
+  Delete "$INSTDIR\controllers\Gemini CDMP-7000 R audio.midi.xml"
+  Delete "$INSTDIR\controllers\Gemini_CDMP-7000-scripts.js"
```
Comment (rryan) [inline build/nsis/Mixxx.nsi]: still has the underscore

**Comment by rryan** [comment]
lgtm (other than the nsis), thanks!

**Comment by Be-ing** [comment]
Why did you include the Gemini CDMP 7000 mapping? It clearly has fixable bugs and unmapped parts and does not use 2.0 features. The author posted 2 weeks ago that they're working on it and have the files on GitHub, indicating that they're capable of making a pull request themselves. Why didn't you ask the author to submit it themselves before doing it for them?

**Pull request merged**